events/events.go

```diff
@@ -9,11 +9,12 @@ import (
 )
 
 const (
-	ReasonStarted   string = "Started"
-	ReasonStarting  string = "Starting"
-	ReasonStopped   string = "Stopped"
-	ReasonStopping  string = "Stopping"
-	ReasonSyncError string = "SyncError"
+	ReasonStarted     string = "Started"
+	ReasonStarting    string = "Starting"
+	ReasonStopped     string = "Stopped"
+	ReasonStopping    string = "Stopping"
+	ReasonSyncError   string = "SyncError"
+	ReasonCreateError string = "CreateError"
```

Comment:
not sure about name here (`CreateError`) maybe better `CreationError` or `InitError`